src/ir/module-utils.h

```diff
@@ -399,6 +395,63 @@ template<typename T> struct CallGraphPropertyAnalysis {
   }
 };
 
+// Helper function for collecting the type signature used in a module
+//
+// Used when emitting or printing a module to give signatures canonical
+// indices. Signatures are sorted in order of decreasing frequency to minize the
```

Comment:
Not sure if `minize` is a typo, or intentional...

Reply:
Definitely a typo!